requirements/edx/base.txt

```diff
@@ -115,8 +116,7 @@ edx-completion==0.1.7
 edx-django-oauth2-provider==1.2.5
 edx-django-release-util==0.3.1
 edx-django-sites-extensions==2.3.1
-edx-drf-extensions==1.4.0
-edx-enterprise==0.69.4
+edx-enterprise==0.69.5
```

Comment:
Did you ask about this?  You should **not** need to ask on this level of change, but the completion api versioning was done incorrectly and it broke things to take the upgrade, so I'm over-sensitive for now. ;)

Reply:
No longer an issue, since someone already upgraded this to `0.70.1`.